docs/sources/tempo/metrics-generator/_index.md

```diff
@@ -46,3 +46,11 @@ Dimensions can be the service name, the operation, the span kind, the status cod
 The more dimensions are enabled, the higher the cardinality of the generated metrics.
 
 To learn more about this processor, read the [documentation]({{< relref "span_metrics/" >}}).
+
+### Exporting metrics
```

Comment:
What do you think about renaming this to "remote writing metrics"? Exporting makes it seem as if they are stored long-term in the metrics-generator.

I don't feel particularly strongly about this though.

Reply:
Done!